test/e2e/bundle_e2e_test.go

```diff
@@ -47,6 +118,30 @@ var _ = Describe("Installing bundles with new object types", func() {
 				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
 			)
 
+			// create VPA CRD on cluster
+			data, err := yaml.YAMLToJSON([]byte(vpaCRDManifest))
```

Comment:
This should all probably go in a `BeforeEach`, and I think we'll also want to delete this specific CRD in `AfterEach` too.